packages/grafana-ui/src/components/BigValue/PercentChange.tsx

```diff
@@ -8,8 +8,7 @@ export interface Props {
 }
 
 export const PercentChange = ({ percentChange, styles }: Props) => {
-  const percentChangeIcon =
-    percentChange && (percentChange > 0 ? 'arrow-up' : percentChange < 0 ? 'arrow-down' : undefined);
+  const percentChangeIcon = percentChange > 0 ? 'arrow-up' : percentChange < 0 ? 'arrow-down' : undefined;
```

Comment:
I'd split this out into an `if... else` rather than having nested ternary operators